core-services/prow/02_config/opendatahub-io/odh-dashboard/_prowconfig.yaml

```diff
@@ -2,6 +2,9 @@ tide:
   merge_method:
     opendatahub-io/odh-dashboard: merge
   queries:
+  - includedBranches:
+    - incubation
+    - main
   - labels:
```

Comment:
Small typo.  You need to remove the list identifier from `labels` since you inserted `- includedBranches` at the top

Reply:
oh my, didn't catch that, thanks @LaVLaS !!